controllers/sriovoperatorconfig_controller_test.go

```diff
@@ -327,5 +328,34 @@ var _ = Describe("SriovOperatorConfig controller", Ordered, func() {
 			})
 			Expect(err).ToNot(HaveOccurred())
 		})
+		It("should deploy the metrics-exporter when the feature gate is enabled", func() {
```

Comment:
do we want to add test case for the delete flow ? (i.e turning off the feature gate after it was enabled)